utils/utils.jsx

```diff
@@ -1339,6 +1340,17 @@ export function imageURLForUser(userIdOrObject) {
     return Client4.getUsersRoute() + '/' + userIdOrObject.id + '/image?_=' + (userIdOrObject.last_picture_update || 0);
 }
 
+export function iconImageURLForUser(userIdOrObject) {
+    if (typeof userIdOrObject == 'string') {
+        const profile = getUser(store.getState(), userIdOrObject);
```

Comment:
There's still a lot of code that uses the global store, but this is considered deprecated as it introduces potential re-rendering issues (the connector won't trigger a re-render for any state queries that occur here).

Can we refactor this to accept the `state` object directly? Also, unless this has strong reuse, I might suggest just defining this function in the connector for the one component that uses it. Easy to refactor out later.

Reply:
Ripped getting user through `getUser` out from here since i really didn't to pass in a `botId`, but rather always a `botuser`.